server/connectors/slack_connector/client.py

```
@@ -112,107 +139,76 @@ def invite_to_channel(self, channel: str, users: List[str]) -> Optional[Dict[str
+        ]
+        
+        channel = None
+        channel_name = None
+        for name in candidates:
+            channel = _try_create_channel(client, name)
+            if channel:
+                channel_name = name
+                break
+        
+        if not channel or not channel_name:
+            logger.error(f"[{team_name}] Failed to create any incidents channel variant")
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)